tests/service/ExprTests.fs

```diff
@@ -991,6 +991,7 @@ let testOperators dnName fsName excludedTests expectedUnoptimized expectedOptimi
     let projFilePath = Utils.getTempFilePathChangeExt tempFileName ".fsproj"
 
     try
+        System.Diagnostics.Debugger.Break()
```

Comment:
What's this?

Reply:
carelessly left over rubbish from debugging.